### ISSUE TITLE
Go to blame view

### ISSUE THREAD

**Issue by CallumHoward** [ruifm/gitlinker.nvim#30]
**Is your feature request related to a problem? Please describe.**
Sometimes I want to copy the 

**Describe the solution you'd like**
I would like an additional mapping to copy (and also one to open in browser) the URL for the Blame view in GitHub, BitBucket etc.

Normal view:
```
https://github.com/ruifm/gitlinker.nvim/blob/master/doc/gitlinker.txt#L53
```

Blame view:
```
https://github.com/ruifm/gitlinker.nvim/blame/master/doc/gitlinker.txt#L53
```

**Describe alternatives you've considered**
- Current work around is to open in browser, click blame and then copy the url
  - or slightly faster with GitHub and browser keybindings: B<C-l><C-c>
- Previously when using Gina I had a mapping that made this possible using `--scheme` flag:
```
nnoremap <silent> <leader>gB V:Gina browse : --scheme=blame<CR>
```

**Additional context**

**Comment by ruifm** [comment]
Good idea, it seems simple to implement through an option, I will work on it this week together with https://github.com/ruifm/gitlinker.nvim/issues/29 in other to make mapping different actions more manageable.

**Comment by CallumHoward** [comment]
Hi, any chance of revisiting this?

**Comment by linrongbin16** [comment]
hi @CallumHoward, let's track this issue here: https://github.com/linrongbin16/gitlinker.nvim
